tools/pixel.md

```diff
@@ -3,51 +3,60 @@ layout: post
 permalink: /pixel
 title: Google Pixel
 icon_slug: google
+command: "Settings -> About Phone -> Regulatory labels"
 link: https://support.google.com/nexus/answer/4457705?hl=en
 discontinuedColumn: true
 activeSupportColumn: false
 latestColumn: true
 eolColumn: Software Update (Guaranteed)
 releaseColumn: false
 releaseDateColumn: true
-sortReleasesBy: 'release'
+sortReleasesBy: 'cycleShortHand'
 releases:
-  - releaseCycle: Pixel 5
+  - releaseCycle: "Pixel 5"
+    cycleShortHand: 500
     release: 2020-09-30
     discontinued: false
     eol: 2023-10-31
     support: 2015-11-19
-  - releaseCycle: Pixel 4A 5G
+  - releaseCycle: "Pixel 4A 5G"
+    cycleShortHand: 420
```

Comment:
Just a quersion: What's the `cycleShortHand` field for?

Reply:
@Kazzarra whenever you name releaseCycles like 1.5 and other like 1.12 for example  because of math it orders it
like 
1.5
1.12
( because 1.5 > 1.12 in math )
so we add another field and and give numbers like 105 for 1.5 and 112 for 1.12 that makes it
order correctly

Reply:
and for here it orders pixel 4A 5G shown in higher than pixel 5
as you can see here : 
![image](https://user-images.githubusercontent.com/112882/126447416-636fc95d-380a-46a5-9503-b0f0cf10ad49.png)

Reply:
As you can see right now in correct order : 
![image](https://user-images.githubusercontent.com/112882/126448028-6a93d5ce-70d1-4696-84d2-1a113e0cadfb.png)